[2025-06-17T10:23:39.422+0900] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-17T10:23:39.483+0900] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: image_ocr_v2.check_file_exists_branch manual__2025-06-17T01:23:38.033833+00:00 [queued]>
[2025-06-17T10:23:39.516+0900] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: image_ocr_v2.check_file_exists_branch manual__2025-06-17T01:23:38.033833+00:00 [queued]>
[2025-06-17T10:23:39.518+0900] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-06-17T10:23:39.568+0900] {taskinstance.py:2890} INFO - Executing <Task(_BranchPythonDecoratedOperator): check_file_exists_branch> on 2025-06-17 01:23:38.033833+00:00
[2025-06-17T10:23:39.591+0900] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=782) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-17T10:23:39.597+0900] {standard_task_runner.py:72} INFO - Started process 791 to run task
[2025-06-17T10:23:39.600+0900] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'image_ocr_v2', 'check_file_exists_branch', 'manual__2025-06-17T01:23:38.033833+00:00', '--job-id', '788', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpe9dmhfx6']
[2025-06-17T10:23:39.609+0900] {standard_task_runner.py:105} INFO - Job 788: Subtask check_file_exists_branch
[2025-06-17T10:23:39.783+0900] {task_command.py:467} INFO - Running <TaskInstance: image_ocr_v2.check_file_exists_branch manual__2025-06-17T01:23:38.033833+00:00 [running]> on host f6d4228a9eb8
[2025-06-17T10:23:40.120+0900] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='image_ocr_v2' AIRFLOW_CTX_TASK_ID='check_file_exists_branch' AIRFLOW_CTX_EXECUTION_DATE='2025-06-17T01:23:38.033833+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-17T01:23:38.033833+00:00'
[2025-06-17T10:23:40.126+0900] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-17T10:23:40.142+0900] {python.py:240} INFO - Done. Returned value was: get_file_info_list_task
[2025-06-17T10:23:40.144+0900] {branch.py:38} INFO - Branch into get_file_info_list_task
[2025-06-17T10:23:40.147+0900] {skipmixin.py:233} INFO - Following branch ('get_file_info_list_task',)
[2025-06-17T10:23:40.188+0900] {skipmixin.py:281} INFO - Skipping tasks [('end_task', -1)]
[2025-06-17T10:23:40.307+0900] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-17T10:23:40.309+0900] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=image_ocr_v2, task_id=check_file_exists_branch, run_id=manual__2025-06-17T01:23:38.033833+00:00, execution_date=20250617T012338, start_date=20250617T012339, end_date=20250617T012340
[2025-06-17T10:23:40.445+0900] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-17T10:23:40.596+0900] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-17T10:23:40.604+0900] {local_task_job_runner.py:245} INFO - ::endgroup::
